homeassistant/components/demo/strings.json

```diff
@@ -63,6 +63,29 @@
     }
   },
   "entity": {
+    "climate": {
+      "ubercool": {
+        "attribute": {
```

Comment:
So, wasn't the consensus above that this should be renamed to `state_attributes` as per frontend usage and consistency with #77028?

Reply:
Yeah, it sure was, I'm not sure how I missed that.